File: DavidVancl/PRJC/Exercise 6/Source/Person.cpp
#include "Person.h"

void fillPerson(Person* p,int id, char setName[],int age) {
	p->id = id;
	p->age = age;
	p->name = (char*)malloc(strlen(setName) * sizeof(char));
	if (p->name == NULL) return;
	strcpy(p->name, setName);
}

void addCar(Person* p, Car c) {
	Car* realloc_garage = (Car*)realloc(p->garage,(p->carCount + 1) * sizeof(Car));
	if (realloc_garage == NULL) return;
	p->garage = realloc_garage;
	p->garage[p->carCount] = c;
	p->carCount++;
}

File: DavidVancl/VAPW/CV05-07/CV06/Person.cs
namespace CV06 {
    class Person {
        private string name;
        public Person(string name) {
            this.name = name;
        }
        public string getName() {
            return this.name;
        }
        public override string ToString() {
            return "Name: " + this.getName();
        }
    }
}
